deepomatic/api/exceptions.py

```diff
@@ -83,11 +84,23 @@ def get_task_id(self):
 ###############################################################################
 
 class TaskTimeout(DeepomaticException):
-    def __init__(self, task):
+    def __init__(self, task, retry_error=None):
```

Comment:
I prefer to avoid inheritance (especially multiple) whenever possible. `retry_error` and `TaskRetryError` is an internal behavior, and is just provided for debug purpose.